apps/prairielearn/elements/pl-file-upload/pl-file-upload.py

```diff
@@ -1,15 +1,23 @@
 import csv
 import hashlib
+import itertools
 import json
+import re
 from io import StringIO
 
 import chevron
 import lxml.html
 import prairielearn as pl
 from colors import PLColor
 
+ALLOW_BLANK_DEFAULT = False
 
+
+# Convert a comma-separated list of file names into an array
```

Comment:
The style in Python is to have comments like this be a docstring on the function:

https://www.programiz.com/python-programming/docstrings

Reply:
I've updated this for this file, and generally agree, but I was trying to stay consistent with the existing documentation/comment style, which is not using docstrings in virtually any of the Python code.

Reply:
Understandable, there's a few things like this where we're trying to change the convention to something more modern.